scene/gui/popup_menu.cpp

```diff
@@ -396,6 +398,11 @@ void PopupMenu::gui_input(const Ref<InputEvent> &p_event) {
 	Ref<InputEventMouseMotion> m = p_event;
 
 	if (m.is_valid()) {
+		if (m->get_velocity().is_equal_approx(Vector2())) {
```

Comment:
Can use `is_zero_approx`.

Reply:
There's no `is_zero_approx` for `Vector2`, only for float/double.